TKGame/BackEnd/EntityManager.cs

```diff
@@ -136,13 +139,44 @@ public static void DrawHealth(SpriteBatch spriteBatch, Entity entity)
         /// </summary>
         public static void DamageEnemy()
         {
-            foreach (Entity entity in entities)
+            int count = 0;
+            foreach(Entity entity in entities)
             {
-                if (entities[0].entityTexture == Art.PlayerTexture) //Checks if player has sword out
+                if(entity.isEnemy && entity.hitBox.Intersects(entities[0].weapon.hitbox))
                 {
-                    if (entities[0].weapon.hitbox.Intersects(entity.hitBox) && entity != entities[0]) //Checks if hitboxes intersect and is not the player
+                    if (count % 240 == 0)
                     {
-                        entity.health -= (int)(entities[0].weapon.damageStat);
+                        if (entity == entities[1])
+                        {
+                            entities[1].health -= entities[0].weapon.damageStat;
+                        }
+                        else if(entity == entities[2])
+                        {
+                            entities[2].health -= entities[0].weapon.damageStat;
+                        }
+                        else if(entity == entities[3])
+                        {
+                            entities[3].health -= entities[0].weapon.damageStat;
+                        }
```

Comment:
uh, why was this added? This looks like you can now only damage the first 3 entities that exist? Also, this is just a for loop transformed into a bunch of if elses. Confused about this change.